vc_zoom/pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
```

Comment:
i changed some stuff in the pytest.ini files of other plugins in master (added our usual filterwarnings config) - maybe use that one already

Reply:
Where exactly? In this repo?